cmake/Modules/CopyMSVCBins.cmake

```diff
@@ -53,41 +52,7 @@ file(GLOB FFMPEG_BIN_FILES
 
 	"${FFMPEG_swresample_INCLUDE_DIR}/../bin/swresample-*.dll"
 	"${FFMPEG_swresample_INCLUDE_DIR}/../bin${_bin_suffix}/swresample-*.dll"
-	"${FFMPEG_swresample_INCLUDE_DIR}/bin${_bin_suffix}/swresample-*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libopus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/opus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libopus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/opus*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libogg*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libvorbis*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libogg*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libvorbis*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libvpx*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libvpx*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libsrt*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libsrt*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libmbedcrypto*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libmbedcrypto*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin${_bin_suffix}/libopus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin${_bin_suffix}/opus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin${_bin_suffix}/libopus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin${_bin_suffix}/opus*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/libbz2*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin/zlib*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/libbz2*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin/zlib*.dll"
-
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin${_bin_suffix}/libbz2*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/../bin${_bin_suffix}/zlib*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin${_bin_suffix}/libbz2*.dll"
-	"${FFMPEG_avcodec_INCLUDE_DIR}/bin${_bin_suffix}/zlib*.dll"
```

Comment:
similar to the way the other non-ilos builds settings are being set, should we include this stuff if it's not an ilos build?

Reply:
Could potentially do that. I think the ILOS_BUILD flag is relatively pointless as of now. It is probably better to just maintain our own branch and then if we are doing a non ilos build just use the obs branch